job-descriptions/internal-tools.md

```diff
@@ -0,0 +1,43 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Internal Tools
+
+We are looking for an engineer and operator to build and improve our internal tools. As we grow our team and customer base rapidly, we need your help building, upgrading, and maintaining the internal systems that are straining to keep up!
+
+## Responsibilities
+
+You will be a key part of improving a wide range of internal (and external) systems and processes. Example projects include:
+
+- Evaluating and implementing single sign-on software to make onboarding smoother.
```

Comment:
I think more details and specifics on the projects would be helpful for sorcerers and for external folks. Can you link to the onboarding handbook page that would be simplified with SSO? Can you give an example of tools/bots (such as the one that notifies when a customer exceeds a threshold in the contract)?